site/content/docs/5.3/examples/form/index.html

```diff
@@ -93,7 +93,7 @@ <h1 class="display-1">Become a reseller</h1>
               </p>
             </div>
```

Comment:
What would you think about make this div `aria-hidden="true"` since it doesn't bring much info to AT users.

Reply:
I am not in favor of it.
The goal here is to give the information to users before they encounter this particular symbol so it's a part of a global 'error prevention strategy'

Reply:
I'm fine with it, my only concern here is: if I'm a user running a not so recent AT that doesn't read CSS content, the AT announce `star indates required` and then I only hear `required` everywhere, so why would I need the first information ? On the other side, the recent ATs read CSS and so `label required star` and if we avoid the first information it could seem weird. I don't know if this is understandable,don't hesitate to ping me if not.